src/config/global-config.h

```diff
@@ -19,6 +19,8 @@ typedef struct __Config {
     u_int8_t stats;
     u_int8_t integrity;
     u_int8_t syscheck_auto_ignore;
+    u_int8_t syscheck_ignore_frecuency;
```

Comment:
Since you are using `atoi()` to assign the values, I recommend you to use `int` simply. Then you should change the condition above (`< 0`) by `<= 0`.